### DISCUSSION TITLE
Sprawdzanie przyczyn endedBy ADMIN

### DISCUSSION THREAD

**Discussion by tomylee0** [allegro/allegro-api#7260] [Suggestion - verification]
Chciałbym ponownie zapytać czy jest już może jakaś możliwość sprawdzenia przez REST API z jakiego powodu oferta ma publication.endedBy = ADMIN. Jest to o tyle istotne, że w zależności od przyczyny musimy się inaczej zachować. Jeśli przyczyną jest:

1. niezgodność oferty z regulaminem to nie powinniśmy więcej tej oferty wystawiać,
2. zduplikowanie oferty na kontach klienta to powinniśmy przeorganizować konta,
3. niezapłacona faktura to po zapłaceniu powinniśmy wystawić na nowo oferty.

Do niedawna oferty endedBy ADMIN z powodu niezgodności z regulaminem pojawiały się sporadycznie. Ostatnio zauważyliśmy, że najprawdopodobniej testujecie wykrywanie adresów www na zdjęciach produktów. A niektórzy producenci umieszczają tam swoje adresy. Dostaliśmy wtedy spore ilości endedBy ADMIN bez dedykowanej możliwości sprawdzenia przyczyny. Wysyłanie maili z informacją o usunięciu oferty przy kontach z wielotysięczną ilością ofert to w mojej ocenie jednak za mało.

Bez dostępu do tej informacji przez REST API ryzykujemy zawieszeniem kont.

**Comment by AureliuszBrussy** [comment]
Wciąż nie udostępniamy możliwości sprawdzenia powodu zakończenia oferty przez admina. Przekonwertowałem równocześnie Twoje zgłoszenie do Discussions, tak, aby inni użytkownicy widzieli tą potrzebę i mogli na nią zagłosować. Przy podwyższonym zainteresowaniu mamy ważny argument do wdrażania takiej sugestii.

**Comment by tomylee0** [comment]
Ok. Dzięki.
